app/src/main/res/layout/fragment_contributions_suggested_edits.xml

```diff
@@ -1,39 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.swiperefreshlayout.widget.SwipeRefreshLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/swipeRefreshLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="?attr/paper_color">
 
-    <LinearLayout
-        android:id="@+id/contentContainer"
+    <FrameLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="?attr/paper_color"
-        android:orientation="vertical">
+        android:layout_height="match_parent">
 
         <androidx.recyclerview.widget.RecyclerView
             android:id="@+id/contributionsRecyclerView"
             android:layout_width="match_parent"
-            android:layout_height="0dp"
-            android:layout_weight="1"
-            android:clipToPadding="false"
-            android:paddingBottom="16dp"
+            android:layout_height="match_parent"
             android:scrollbars="vertical" />
 
         <ProgressBar
-            android:id="@+id/loadMoreProgressView"
+            android:id="@+id/progressBar"
```

Comment:
I remember Robin wanting a progress indicator at the bottom, to indicate lazy loading, like the explore feed. This will probably be ok, but might have to check with him.